packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -519,9 +531,31 @@ class CupertinoPopupSurface extends StatelessWidget {
   /// Creates an iOS-style rounded rectangle popup surface.
   const CupertinoPopupSurface({
     super.key,
+    this.blurSigma = defaultBlurSigma,
     this.isSurfacePainted = true,
-    this.child,
-  });
+    this.isVibrancePainted = true,
+    required this.child,
+  }) : assert(blurSigma >= 0);
+
+  /// The strength of the gaussian blur applied to the area beneath this
+  /// surface.
+  ///
+  /// Defaults to [defaultBlurSigma]. Setting [blurSigma] to 0 will remove the
+  /// blur filter.
+  final double blurSigma;
+
+  /// Whether or not the area beneath this surface should be saturated with a
+  /// [ColorFilter].
+  ///
+  /// The appearance of the [ColorFilter] is determined by the [Brightness]
+  /// value obtained from the ambient [CupertinoTheme].
+  ///
+  /// When stacking multiple [CupertinoPopupSurface]s, setting
+  /// [isVibrancePainted] to false can prevent the backdrop from becoming too
+  /// saturated.
+  ///
+  /// Defaults to true.
+  final bool isVibrancePainted;
```

Comment:
If it turns out only HTML renderer is not supported, we might want to turn `isVibrancePainted` into `debugIsVibrancePainted` because I can't see it being very useful for production purposes for now.

Reply:
Changed!